iroha/src/torii/mod.rs

```diff
@@ -138,6 +139,17 @@ impl<W: WorldTrait> Torii<W> {
     /// Can fail due to listening to network or if http server fails
     #[iroha_futures::telemetry_future]
     pub async fn start(self) -> iroha_error::Result<()> {
+        #[allow(clippy::unused_async)]
+        async fn recover(err: Rejection) -> Result<impl Reply, Rejection> {
+            if let Some(err) = err.find::<AcceptQueryError>() {
+                return Ok(reply::with_status(err.to_string(), err.status_code()));
+            }
+            if let Some(err) = err.find::<iroha_version::error::Error>() {
+                return Ok(reply::with_status(err.to_string(), err.status_code()));
+            }
```

Comment:
Do I understand correctly that this are errors for which we return custom Error http codes?

Reply:
Thats all errors in parsing arguments